sdk/communication/AzureCommunicationChat/Source/Generated/Options/AzureCommunicationChatClientOptionsInternal.swift

```diff
@@ -15,31 +15,31 @@ import Foundation
 // swiftlint:disable line_length
 
 /// User-configurable client options.
-public struct AzureCommunicationChatClientOptions: ClientOptions {
```

Comment:
What's the difference between this model and the public version? If it's just the errorHandler you can just set it to nil for the internal usage.

Reply:
Thats correct its just the error handler. was trying to avoid modifying the generated code, but probably a better way. Is there a way I can get autorest to add the error handler to the options?

Reply:
^AzureCommunicationChatClientOptions is autogenerated so I was trying to avoid manually adding error handler in the generated code

Reply:
That makes sense and no, you can't add anything to the generated class.

However, you could make everything use just the public class and get rid of the internal class entirely.

Reply:
I think the issue is the generated options are used by other generated code (ChatClientInternal), so I can't remove it completely. next time the code is generated we'd have to keep deleting the generated version